homeassistant/components/qbittorrent/__init__.py

```diff
@@ -35,11 +35,9 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
             entry.data[CONF_VERIFY_SSL],
         )
     except LoginRequired as err:
-        _LOGGER.error("Invalid credentials")
-        raise ConfigEntryNotReady from err
+        raise ConfigEntryNotReady("Invalid credentials") from err
```

Comment:
Note: is there a plan to implement reauth?

Reply:
Not at this moment, but I'll look into it when I have more time.